config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigtableappprofiles.bigtable.cnrm.cloud.google.com.yaml

```diff
@@ -162,26 +176,23 @@ spec:
                       type: string
                   type: object
                 type: array
-              name:
```

Comment:
Ah, it looks like "name" might be an output field. Let's move it from spec to status in our appprofile_types.go